src/main.rs

```diff
@@ -51,6 +51,12 @@ fn main() {
                 .takes_value(true)
                 .min_values(1),
         )
+        .arg(
+            Arg::with_name("disable-console")
```

Comment:
I think we should always enable `virtio-console`. This means the input (stdin) should be sent through `virtio-console` instead of legacy serial. And we should keep legacy serial only for stdout as we still want to be able to see all kernel logs from the beginning.
Therefore, I don't think we need this option.

I think one thing we want to do longer term is to be able (in the context of production) to compile the serial completely out of our VMM. And by compiling it out, we should also be able to compile the userspace IOAPIC out. This will allow us to reach the minimal amount of emulation for our VMM.

Reply:
This option is removed and replaced with --console <off | hvc | file=/path/to/file> to keep consistent with serial.